cmd/swctl/app/cmd_manage.go

```diff
@@ -76,6 +77,7 @@ func (opts *ManageOptions) InstallFlags(flagset *pflag.FlagSet) {
 	flagset.BoolVar(&opts.ShowConfig, "show-config", false, "Print config for entity detail")
 	flagset.BoolVarP(&opts.Interactive, "interactive", "i", false, "Enable interactive mode")
 	flagset.StringToStringVar(&opts.Vars, "var", nil, "Override values for variables (--var VAR_NAME=value)")
+	flagset.StringVar(&opts.File, "file", "", "Define the output file name (default: <entityName>.yaml)")
```

Comment:
Can you please change the flag to `output-file` (or something similar) as it is not clear whether it is input or output file or what for is given file. We have there also target that is basically input file for some cases (also not that clear). So lets make it clear by using flag names that are almost self-explaining.